### PR TITLE
Made timers daemons to stop blocking exit on quit, set main class

### PR DESCRIPTION
**Pull request by redjen**
I noticed that the app was still running after quit. The cause was that the timers in LoginController and TabViewController were still running. I set these to run as daemons so they will not block exit.

Formatting caused the diff to be rather busy, so I marked the major changes with comments.

Major changes:

1. The timer on the login view is both a daemon and canceled when navigating away (ie successful login). Both are necessary as the user can quit without logging in. I think this may have been preventing the view scene from being garbage-collected in addition to preventing the app from shutting down.

2. Made the timer in TabViewController a daemon.

Minor changes:

1. Main class set in nbproject.properties. 

2. Formatting fixes in the controllers.

### CODE REVIEW

File: src/app/TabViewController.java [hunk 1]
```diff
@@ -26,63 +26,75 @@
...
-            }
-        }, 0, 60 * 1000);
-    }
+   /**
+    * Starts the view's clock
+    */
+   private void startClock() {
+      String timeStamp = new SimpleDateFormat("HH:mm").format(Calendar.getInstance().getTime());
+      clockLabel.setText(timeStamp);
+      timer = new java.util.Timer(true);
```
Comment (redjen) [inline src/app/TabViewController.java]: Important change

File: src/app/LoginController.java [hunk 2]
```diff
@@ -23,60 +23,66 @@
...
-            }
-        }, 0, 60 * 1000);
-    }
+   /**
+    * Starts the view's clock
+    */
+   private void startClock() {
+      String timeStamp = new SimpleDateFormat("HH:mm").format(Calendar.getInstance().getTime());
+      clockLabel.setText(timeStamp);
+      timer = new java.util.Timer(true);
```
Comment (redjen) [inline src/app/LoginController.java]: Line 58 is one of the important changes

File: src/app/LoginController.java [hunk 3]
```diff
@@ -23,60 +23,66 @@
...
+    * Login event when login button is clicked.
+    *
+    * @param ActionEvent when button clicked
+    */
+   public void login(ActionEvent event) throws IOException {
+      boolean isLoggedIn = true;
+
+      if (isLoggedIn) {
+         timer.cancel();
+         ViewManager.getManager().navigate("TabView");
```
Comment (redjen) [inline src/app/LoginController.java]: Line 82 is one of the important changes.

File: src/app/LoginController.java (hunk 2 above)
Comment (redjen) [inline src/app/LoginController.java:58]: major change #1

File: src/app/TabViewController.java (hunk 1 above)
Comment (redjen) [inline src/app/TabViewController.java:48]: Major change #2

File: src/app/LoginController.java [hunk 4]
```diff
@@ -23,60 +23,66 @@
...
+   /**
+    * Login event when login button is clicked.
+    *
+    * @param ActionEvent when button clicked
+    */
+   public void login(ActionEvent event) throws IOException {
+      boolean isLoggedIn = true;
+
+      if (isLoggedIn) {
+         timer.cancel();
```
Comment (redjen) [inline src/app/LoginController.java:81]: Major change #1

**Review by maximdumont — APPROVED** [review]

**Pull request merged**